src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const reversedGender = (sex === 'm') ? 'wife' : 'husband';
+
+  return (
+    <div className="Person">
+      <h3 className="Person__name">
+        {`My name is ${name}`}
+      </h3>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
```

Comment:
```suggestion
      {age && (
        <p className="Person__age">
            {`I am ${age}`}
         </p>
      )}
```

Just for consistency on code style, as you wrote it above.